site/app/templates/officeHoursQueue/QueueFooter.twig

```diff
@@ -228,5 +234,18 @@
       });
   }
 
+  function toggleQueue(index, code){
+      console.log($(`#toggle_queue_${index}`)[0].checked)
+      $.post( `{{base_url}}/${code}/toggle`, {
+          csrf_token: '{{ csrf_token }}',
+          queue_code: code,
+          queue_state: $(`#toggle_queue_${index}`)[0].checked ? 0:1
+      });
+  }
+
+  function refreshPage(){
```

Comment:
I don't think we need to wrap this in a function, since its only being used in that one place.
You can just do a `onclick="window.location.reload()"`